_search-plugins/searching-data/filtering-search-results.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: default
+title: Filter results
+parent: Searching data
+nav_order: 21
+redirect_from:
+  - /opensearch/search/filter/
+---
+# Filtering search results in OpenSearch
+
+Filtering the search results in OpenSearch allows you to refine the results returned. Specifying criteria to filter the indexes/documents by, allows you to refine the returned results. You can filter the results based on ranges, conditions or specific terms to refine the returned results. This is helpful when you have large datasets and allows you to interpret and understand larger datasets more easily.
+
+Filtering allows you to:
+- Improve search accuracy by reducing unneeded information on a case by case basis to allow accurate interpretation of results. 
+- Enhance performance by reducing the amount of data necessary to process and return when querying. Improving query performance. 
+- Categorised search by allowing data to be returned in categories which allows you to explore the data in a structured manner.
+
+## Filtering with OpenSearch Dashboards
+
+To begin filtering in OpenSearch Dashboards, follow these steps:
+
+1. Navigate to the OpenSearch Dashboards UI.
+2. Click on `Discover` in the sidebar.
+3. Choose `opensearch_dashboards_sample_data_flights` from the index pattern selector.
+
+### Example: Filter by destination airport
+
+To filter flights arriving at "Zurich Airport":
+
+**Add a filter:**
+- Click on the `Add filter` button.
+- In the filter field, select `Dest`.
+- In the operator field, select `is`.
+- In the value field, type `Zurich Airport`.
+- Click `Save`.
+
+This will display only the flights arriving at Zurich Airport.
+
+### Example: Filter by flight delay
+
+To filter flights that have been cancelled in the last 100 days:
+- Click on the timeframe to update the time selection.
+- Under the 'Relative' time tab.
+- Change the unit to '1'.
+- From the dropdown select 'Days ago'.
+- The data shown is now from the Last 100 days.
+
+**Add a filter:**
+- Click on the `Add filter` button.
+- In the filter field, select `Cancelled`.
+- In the operator field, select `is`.
+- In the value field, type `true`.
+- Click `Save`.
+
+This will display only the flights that have been cancelled with a destination of Zurich Airport in the last 100 days.
+
+## Using Query DSL for advanced filtering
+
+OpenSearch Query DSL (Domain Specific Language) allows for more complex and powerful queries. You can combine multiple conditions and use advanced logic to filter data.
```

Comment:
Link to the Query DSL section here (maybe specifically to https://opensearch.org/docs/latest/query-dsl/query-filter-context/#filter-context and https://opensearch.org/docs/latest/query-dsl/compound/bool/)

Reply:
Added.